pkg/ingestor/api/api.go

```diff
@@ -124,7 +124,7 @@ func (g *IngestorAPI) Ingest(_ context.Context, path string) error {
 	if err != nil {
 		log.I.Warnf("no metadata has been parsed (old dump format from v1.4.0 or below do not embed metadata information): %v", err)
 		// Backward Compatibility: Extracting the metadata from the path
-		dumpMetadata, err := dump.ParsePath(archivePath)
```

Comment:
Not sure to understand why this change it needed, in the unit test iirc you did an example with the `.tar.gz` file

Reply:
because the archivePath is an hardcoden string, so we need to parse the key which contains the clustername and runid